frontend/src/concepts/pipelines/content/useDeleteStatuses.tsx

```diff
@@ -39,9 +40,19 @@ const useDeleteStatuses = ({
     const deleteErrors = deleteResults.reduce<React.ReactNode[]>((acc, state, i) => {
       if (state instanceof Error) {
         const resource = toDeleteResources[i];
+        // TODO: remove this casting when we move all to v2
         acc.push(
-          <p key={resource.name}>
-            <b>{resource.name}</b>: {state.message}
+          <p
+            key={
+              (resource as PipelineCoreResourceKFv2).display_name ??
+              (resource as PipelineCoreResourceKF).name
+            }
+          >
+            <b>
+              {(resource as PipelineCoreResourceKFv2).display_name ??
+                (resource as PipelineCoreResourceKF).name}
```

Comment:
Not that it overly matters -- but next time make a variable lol, they don't cost much.

Reply:
oh this is temp code tho. just wanted to make minimal impact